ixmp/core.py

```diff
@@ -287,7 +287,21 @@ def add_region_synonym(self, region, mapped_to):
         self._backend.set_node(region, synonym=mapped_to)
 
     def timeslices(self):
-        """Return all sub-annual time slices.
+        """Return all subannual timeslices defined in this Platform instance.
+
+        Timeslices are a way to represent subannual temporal resolution in
+        timeseries data. A timeslice consists of a **name** (e.g., 'January',
```

Comment:
nit: we should maybe be consistent about upper/lower/capital case referring timeslices, otherwise wi end up having SUMMER, summer, Summer vales as timeslices

Reply:
good point - I'd go for lower-case wherever possible. updated.